docs/integrations/integration_datahub.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Integrating DataHub With Great Expectations
+authors:
+    name: Mayuri Nehate, John Joyce, Maggie Hays 
+    url: https://datahubproject.io
+---
+
+:::info
+* Maintained By: DataHub
+* Status: Beta
+* Support/Contact: https://slack.datahubproject.io/
+:::
+
+### Introduction
+This integration allows you to push the results of running expectations into DataHub (https://datahubproject.io/). DataHub is a metadata platform which enables search & discovery, federated governance, and data observability for the Modern Data Stack.
```

Comment:
```suggestion
This integration allows you to push the results of running Expectations into DataHub (https://datahubproject.io/). DataHub is a metadata platform which enables search & discovery, federated governance, and data observability for the Modern Data Stack.
```
As part of our doc standards, we want to make sure that we title case anything that is treated as a technical term -- basically, anything that corresponds to an entry in our [glossary of terms](https://docs.greatexpectations.io/docs/glossary).